content/20.pharmaceuticals.md

```diff
@@ -139,14 +156,20 @@ Other analyses have supported the importance of disease course in determining th
+However, the exact outcomes of such treatments are difficult to predict _a priori_, and there are several examples where early enthusiasm was not borne out in subsequent trials.
+One of the most famous examples analyses of whether a well-known medication could provide benefits to COVID-19 patients came from the assessment of chloroquine (CQ) and hydroxychloroquine (HCQ), which are used for the treatment and prophylaxis of malaria as well as the treatment of lupus erythematosus and rheumatoid arthritis in adults [@dailymed-hcq].
 These drugs are lysosomotropic agents, meaning they are weak bases that can pass through the plasma membrane, and it was thought that they might provide benefits against SARS-CoV-2 by interfering with the digestion of antigens within the lysosome and inhibiting CD4 T-cell stimulation while promoting the stimulation of CD8 T-cells [@doi:10.1111/j.1529-8019.2007.00131.x].
-These compounds also have anti-inflammatory [@doi:10.1111/j.1529-8019.2007.00131.x] and can decrease the production of certain key cytokines involved in the immune response, including interleukin-6 (IL-6), and inhibit the stimulation of Toll-like receptors (TLR) and TLR signaling [@doi:10.1111/j.1529-8019.2007.00131.x].
+These compounds also have anti-inflammatory properties [@doi:10.1111/j.1529-8019.2007.00131.x] and can decrease the production of certain key cytokines involved in the immune response, including interleukin-6 (IL-6) and inhibit the stimulation of Toll-like receptors (TLR) and TLR signaling [@doi:10.1111/j.1529-8019.2007.00131.x].
+
 _In vitro_ analyses reported that CQ inhibited cell entry of SARS-CoV-1 [@doi:10.1186/1743-422X-2-69] and that both CQ and HCQ inhibited viral replication within cultured cells [@doi:10.1093/cid/ciaa237], leading to early hope that it might provide similar therapeutic or protective effects in patients.
 However, while the first publication on the clinical application of these compounds to the inpatient treatment of COVID-19 was very positive [@doi:10.1016/j.ijantimicag.2020.105949], it was quickly discredited [@url:https://www.isac.world/news-and-publications/official-isac-statement].
-Over the following months, extensive evidence emerged demonstrating that CQ and HCQ offered no benefits for COVID-19 patients and, in fact, carried the risk of dangerous side effects (see Appendix XX).
+Over the following months, extensive evidence emerged demonstrating that CQ and HCQ offered no benefits for COVID-19 patients and, in fact, carried the risk of dangerous side effects (Appendix A).
 The nail in the coffin came when findings from the large-scale RECOVERY trial were released on October 8, 2020.
 This study enrolled 11,197 hospitalized patients whose physicians believed it would not harm them to participate and used a randomized, open-label design to study the effects of HCQ compared to SOC at 176 hospitals in the United Kingdom [@doi:10.1056/NEJMoa2022926].
```

Comment:
```suggestion
This study enrolled 11,197 hospitalized patients whose physicians believed it would not harm them to participate and used a randomized, open-label design to study the effects of HCQ compared to standard of care (SOC) at 176 hospitals in the United Kingdom [@doi:10.1056/NEJMoa2022926].
```